pkg/cache/cache.go

```diff
@@ -64,9 +65,6 @@ func GetCacheFunc() (cache.NewCacheFunc, error) {
 		&networkingv1.Ingress{}: {
 			Label: devworkspaceObjectSelector,
 		},
-		&routev1.Route{}: {
```

Comment:
I saw that on the first commit and started minishift to check it out but now I don't need it :-D